docs/reference/template-tags.md

```diff
@@ -0,0 +1,191 @@
+| [`component`](#component) | Insert an [async component](/guide/async-components.md) | Yes |
+| [`fragment`](#fragment) | Declare a [template fragment](/guide/fragments.md) | No |
+| [`render`](#render) | Insert a basic template fragment | Yes |
+| [`rendercall`](#rendercall) | Insert a template fragment that includes a `rendercaller` slot | No |
+| [`widget`](#widget) | Used in the core area template to render individual widgets | Yes |
+
+
+## `area`
+
+The `area` tag inserts an area field into the template. The area field [must already be configured](/guide/areas-and-widgets.md#basic-area-configuration) in the page
+
+### Usage
+
+```django
+{% area context, areaName with contextOptions %}
+```
+
+**Example:**
+```django
+{% area data.page, 'main' with {
+  '@apostrophecms/image': {
+    sizes: '(min-width: 600px) 45vw, (min-width: 1140px) 530px'
+  }
+} %}
+```
+
+### Arguments
+
+#### `context`
+
+The area's document context, either a page (`data.page`), piece (`data.piece`), or widget (`data.widget`). The area field must be defined in the field schema for that context. See the [template data](/guide/template-data.md) section for more on each `data` property.
+
+#### `areaName`
+
+The name (a string) of the area field as defined in the field schema.
+
+#### `contextOptions` (optional)
+
+The context options object is added after area tag arguments following the `with` keyword. It is an object with keys matching the names of widget types allowed in that particular area field. Each key is assigned a value that will be passed into the widget template as `data.contextOptions`. See the [image widget guide](/guide/core-widgets.md#image-widget) for an example.
+
+Context options are optional for all core and official Apostrophe widget types.
+
+## component
+
+The `component` tag inserts an asynchronous component into the template. See the [async components guide](/guide/async-components.md) for more on using this feature.
+
+### Usage
+
+```django
+{% component 'moduleName:componentName' with data %}
+```
+
+**Example:**
+```django
+{% component 'product:newest' with { max: 3 } %}
+```
+
+### Arguments
+
+#### `moduleName:componentName`
+
+The primary argument is a combination of the name of a module name and the name of an async component from that module, separated by a colon. All async components belong to a specific module, though multiple modules may have components with the same name.
+
+#### `data` (optional)
+
+The data argument, following the `with` keyword, is available in the async component template as `data`. It can be any data type, however it is a best practice to make it an object with subproperties so using the values are clearer in templates.
+
+## fragment
+
+The `fragment` tag *declares* a template fragment that will be inserted elsewhere with [`render`](#render) or [`rendercall`](#rendercall). See the [template fragments guide](/guide/fragments.md) for more on using this feature.
```

Comment:
We say that very explicitly in the fragments guide, which I linked to there. I want to avoid additional diversions in the reference section as much as possible.